controllers/core/pod_controller_test.go

```diff
@@ -16,19 +16,19 @@ package controllers
 import (
 	"testing"
 
-	mock_controller "github.com/aws/amazon-vpc-resource-controller-k8s/mocks/amazon-vcp-resource-controller-k8s/controllers/custom"
-	mock_handler "github.com/aws/amazon-vpc-resource-controller-k8s/mocks/amazon-vcp-resource-controller-k8s/pkg/handler"
+	"github.com/aws/amazon-vpc-resource-controller-k8s/controllers/custom"
+	mockhandler "github.com/aws/amazon-vpc-resource-controller-k8s/mocks/amazon-vcp-resource-controller-k8s/pkg/handler"
 	mock_node "github.com/aws/amazon-vpc-resource-controller-k8s/mocks/amazon-vcp-resource-controller-k8s/pkg/node"
-	"github.com/aws/amazon-vpc-resource-controller-k8s/pkg/handler"
+	mock_resource "github.com/aws/amazon-vpc-resource-controller-k8s/mocks/amazon-vcp-resource-controller-k8s/pkg/resource"
```

Comment:
Still a minor inconsistency here. `mockhandler` vs `mock_resource`

Reply:
these comments could be addressed in next PR. This is approved and can be merged.

Reply:
Will approve in the next PR. Thanks!